pkgs/servers/home-assistant/appdaemon.nix

```diff
@@ -82,6 +82,6 @@ in python.pkgs.buildPythonApplication rec {
     description = "Sandboxed Python execution environment for writing automation apps for Home Assistant";
     homepage = "https://github.com/AppDaemon/appdaemon";
     license = licenses.mit;
-    maintainers = with maintainers; [ peterhoeg ];
+    maintainers = with maintainers; [ peterhoeg ] ++ teams.home-assistant.members;
```

Comment:
@peterhoeg are you still using this?

Reply:
I am not - I moved everything over to `openhab` when HA upstream mentioned that GUI is the way to go. And since openhab upstream is keeping configuration via files a first class citizen (and simply generating all the various data from a single source is awesome) I don't foresee coming back to HA.

Reply:
Addressed in [`7935fd4` (#118633)](https://github.com/NixOS/nixpkgs/pull/118633/commits/7935fd4fbb41e6e19dbcef06ef1a14556386df30). Thanks!

Reply:
What a bummer that OpenHAB is written in Java and probably a PITA to compile from source using Nix.